lagextraction/perform_extraction.m

```diff
@@ -40,7 +40,8 @@
     disp('---- Running lag extraction');
 end
 
-[lags,use_ascend,dc,sc,E] = gc_lags(points_ordered,options);
+%[lags,use_ascend,dc,sc,E] = gc_lags(points_ordered,options);
```

Comment:
no need to keep this commented line. Just remove it and ok for me to merge. thx

Reply:
OK, done. Merci.